CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased](https://github.com/fishbrain/tarpon/compare/v0.2.0...HEAD)
+
+### Added
+- Initial changelog
+
+### Changed
+- Added some useful links to the readme with some minor text adjustments
+
+## [0.2.0](https://github.com/fishbrain/tarpon/compare/v0.1.1...v0.2.0) - 2020-02-24
```

Comment:
The tags are not prefixed with `v` and although not a rule it seems to be the common pattern, at least in version control, according to [semver.org](https://semver.org/#is-v123-a-semantic-version).

I will add those.